test/blackbox-tests/test-cases/formatting/non-ascii-characters.t

```diff
@@ -1,13 +1,30 @@
-How the non-ASCII characters are handled, this is also related to the issue #9728
+Utf8 characters are handled for now, this is also related to the issue #9728
 
   $ dune format-dune-file <<EOF
-  > ("É")
+  > ("Éﬀ ĎúÑȨ")
   > EOF
-  ("\195\137")
+  ("Éﬀ ĎúÑȨ")
 
   $ dune format-dune-file <<EOF
   > (run foo %{bin:é})
   > EOF
   File "", line 1, characters 15-16:
-  Error: The character '\195' is not allowed inside %{...} forms
+  Error: The character é is not allowed inside %{...} forms
+  [1]
+
+  $ dune format-dune-file <<EOF
+  > (echo "hÉllo")
+  > EOF
+  (echo "hÉllo")
+
+  $ dune format-dune-file <<EOF
+  > (echo "É")
+  > EOF
+  (echo "É")
+
+  $ dune format-dune-file <<EOF
+  > (Écho "hello")
+  > EOF
+  File "", line 1, characters 1-1:
+  Error: Invalid . file
```

Comment:
where does the ` . ` part come from?

Reply:
This is coming from the lexer https://github.com/ocaml/dune/blob/1f80d72a2afdf49ff9bd32e340727d1e0babcdb8/src/dune_sexp/lexer.mll#L30
This is because there's no file created with current test. 
Changing it by giving a file, it gives the name of file instead of `.`:
```
$ cat > dune <<EOF
> (Écho "hello")
> EOF
> dune format-dune-file dune
File "$TESTCASE_ROOT/dune", line 1, characters 1-1:
(Écho "hello")

Error: Invalid dune file
[1]
```

Reply:
ok, thanks!